deploy/role.yaml

```diff
@@ -1,5 +1,26 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: Role
+metadata:
+  name: vitess-operator-subcontroller
+  namespace: example
```

Comment:
Isn't this supposed to be in the default namespace?

Reply:
Can you review/fix all the places where this is relevant? Then I can review again.

Reply:
Maybe not the subcontroller if it is meant for backups, but there are tons of places in this PR where vitess-operator is in the example namespace.

Reply:
OK, this is just creating metadata both _both_ namespaces. It wasn't obvious from the diff, but at line 25, the namespace is implicitly default.